app/src/main/java/org/mozilla/focus/state/AppAction.kt

```diff
@@ -51,7 +52,7 @@ sealed class AppAction : Action {
     /**
      * The app should get locked.
      */
-    object Lock : AppAction()
+    data class Lock(val bundle: Bundle? = null) : AppAction()
```

Comment:
Can documentation be added for the new parameter?

Reply:
Done

Reply:
Done